roles/robots_txt/vars/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+robots_txt:
+  user_agents:
+    - {name: 'ScoutJet', crawl_delay: 10}
+    - {name: '*'}
+    - {name: 'Baiduspider', crawl_delay: 10}
+    - {name: 'BecomeBot', crawl_delay: 20}
+    - {name: 'Slurp', crawl_delay: 10}
+  disallow_list:
+    - legacy.cnx.org
+    - /lenses
+    - /browse_content
+    - /content/
+    - /content$
+    - /*/pdf$
+    - /*/epub$
+    - /*/complete$
+    - /*/offline$
+    - /*?format=*$
+    - /*/multimedia$
+    - /*/lens_add?*$
+    - /lens_add
+    - /*/lens_view/*$
+    - /content/*view_mode=statistics$
```

Comment:
I'm not a fan of buried configuration. But maybe I'm looking at this stuff wrong.

Reply:
You mean you would prefer to move this to group vars?  I can also move this to the template (but move the user agent config to group vars)

Reply:
Hmm, there's actually two levels of config here - user-agents to disallow would be universal, but which paths are not-of-interest for crawling is an aspect of the web-apps deployed, so shouldn't those apps configure fragments for robots.txt, like the haproxy config does?

Reply:
So, I think I'm voting "vars" right?

Reply:
These are all legacy urls, right?

Reply:
yes